src/__tests__/config.test.ts

```diff
@@ -5,7 +5,7 @@ describe('configuration', () => {
     const ethrDid = new EthrDID({
       identifier: '0xC662e6c5F91B9FcD22D7FcafC80Cf8b640aed247',
       rpcUrl: 'http://localhost:9585',
-      chainNameOrId: 1337,
+      chainNameOrId: BigInt(1337),
```

Comment:
By the same reasoning as ethr-did-resolver, the chain ID should be string | number.

It's ok to accept bigint, but the primitives should be the default

Reply:
Looking at the code again I think the field can anyhow not be used with a chainId? It is being fed to the EthrDidResolver and used as the chain name to look into the deployment.ts mapping. The tests are all not checking for the id being resolved to a correct chain name. The test "supports numbered chainId" implicitly falls back to the default being mainnet.

Reply:
Changed it to number now again!